stubs/six/six/__init__.pyi

```diff
@@ -1,10 +1,12 @@
 from __future__ import print_function
 
+import builtins
 import types
 import unittest
 from builtins import next as next
 from collections.abc import Callable, ItemsView, Iterable, Iterator as _Iterator, KeysView, Mapping, ValuesView
 from functools import wraps as wraps
+from importlib.util import spec_from_loader as spec_from_loader
```

Comment:
This won't work on Python 2.

Reply:
These stubs are Python 3 only. Python 2 stubs are in the `@python2` directory.